.github/scripts/neo-cli-csharp/dpkg/srv/neo-node/config.fs.mainnet.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
can we use/copy the config files from the source?

Reply:
they are for  the debian packages, They need a custom `storage` path.

Reply:
But it's possible to copy them in the script?

Reply:
I have a script. `build-dpkg.sh` I think the name is. I need these templates. I can look into a way of changing the data for the `config` `json` files.

Reply:
@cschuchardt88, can you copy the files and change them with `sed`?
I think it will be better for maintenance.